server/controllers/reviewsController.js

```diff
@@ -35,3 +35,39 @@ exports.review_get_meta = (req, res) => {
     .then((data) => res.send(data))
     .catch(() => res.sendStatus(500));
 };
+
+exports.review_mark_helpful = (req, res) => {
+  const { review_id } = req.params;
+
+  const config = {
+    headers: { Authorization: process.env.TOKEN },
+  };
+
+  axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sfo/reviews/${review_id}/helpful/`, config)
+    .then(() => res.sendStatus(204))
+    .catch(() => res.sendStatus(500));
+};
+
+exports.review_report = (req, res) => {
+  const { review_id } = req.params;
+
+  const config = {
+    headers: { Authorization: process.env.TOKEN },
```

Comment:
I was trying axios default headers (or make it as a global config const), which avoids create a same config const in each request. However, it's great if you expect different config const to be used for each request.

https://axios-http.com/docs/config_defaults

Reply:
This is a great idea! I'll look into this & update in the future.